source/_integrations/togrill.markdown

```diff
@@ -46,7 +46,20 @@ Many ToGrill compatible devices exist from many different vendors. Only a subset
 
 ## Events
 
-**Probe X**: The most recently triggered alarm or event on the probe. The following event types are expected: `alarm`, `acknowledge`, `disconnected`.
```

Comment:
Noticed that these event_types were actually incorrect. So has been renamed to the real event type below.